commands/accounts/list.ts

```diff
@@ -112,10 +111,6 @@ exports.handler = async options => {
 };
 
 exports.builder = yargs => {
-  addConfigOptions(yargs);
```

Comment:
Whoops, got a little carried away. I'll the config options back

Reply:
https://github.com/HubSpot/hubspot-cli/pull/1265/commits/382facff635ca769953e6a5e1a402f6a56e4a491